Example/NYTPhotoViewer-Swift/ViewController.swift

```diff
@@ -0,0 +1,131 @@
+//
+//  ViewController.swift
+//  NYTPhotoViewer-Swift
+//
+//  Created by Mark Keefe on 3/20/15.
+//  Copyright (c) 2015 The New York Times. All rights reserved.
+//
+
+import UIKit
+
+class ViewController: UIViewController, NYTPhotosViewControllerDelegate {
+
+    @IBOutlet weak var imageButton : UIButton?
+    private let photos = PhotosProvider().photos
+    private var photosViewController: NYTPhotosViewController
```

Comment:
The original intention of the sample project was to mimic how we thought the view controller would be used in most applications. It seems unlikely to me that apps would have a property for a view controller that they will present in the future and that they would initialize that view controller this early, in the initializer. It seems more likely, to me at least, that they would create and instantiate a view controller at the time they need to display it - in this case, when the image is tapped. The Objective-C version of the sample app instantiates the view controller without storing a property when the image is tapped, and I think that it makes a bit more sense and is closer to how people would usually use it in their apps.